composer.json

```diff
@@ -30,7 +30,7 @@
         "ext-ctype": "*",
         "ext-iconv": "*",
         "dantleech/phpcr-migrations-bundle": "^1.0",
-        "doctrine/doctrine-bundle": "^1.10",
+        "doctrine/doctrine-bundle": "^1.12",
```

Comment:
Do we require that update? Here it might be fine, but if we change that in `sulu/sulu` as well (as you are suggesting with your todo in this PR) then this might break some existing applications.